ui/apps/platform/cypress/integration/vulnerabilities/workloadCves/WorkloadCves.helpers.js

```diff
@@ -1,8 +1,18 @@
+import { addDays, format } from 'date-fns';
+import { getDescriptionListGroup } from '../../../helpers/formHelpers';
 import { visit } from '../../../helpers/visit';
 import { selectors } from './WorkloadCves.selectors';
 
 const basePath = '/main/vulnerabilities/workload-cves/';
 
+export function getDateString(date) {
```

Comment:
I think we have a `getDate` from `dateUtils`. Would that be useful here?

Reply:
I had thought about it, but noticed that we don't import any app code into Cypress test files. I'm not sure if this was a technical limitation or an intentional choice but I think it makes sense to keep the test utils separate.